framework/src/transfers/MultiAppScalarToAuxScalarTransfer.C

```diff
@@ -120,7 +120,7 @@ MultiAppScalarToAuxScalarTransfer::execute()
                        "receiving data for the "
                        "MultiAppScalarToAuxScalarTransfer!");
 
-          for (auto j = beginIndex(from_values); j < from_values.size(); ++j)
+          for (unsigned int j = 0; j < from_values.size(); ++j)
```

Comment:
And here as well?

Reply:
Damn you noticed! I need to look at this more. This is a MooseArray and it failed to work with `MooseIndex`. I believe the issue is that it defines `size()`, which has something to do with the wrong template type being selected. I plan to look at this a little more as I'd like to make a sweep through the framework for all loops that start with `for (unsigned` and look for more candidates to change. This can probably go in as is for now, but it's not urgent if you'd prefer it be fixed first. I did check and the right hardcoded type for looping over a `MooseArray` is indeed `unsigned int`.